modules/oadp-1-4-0-release-notes.adoc

```diff
@@ -0,0 +1,66 @@
+With this update, restore works as expected in {product-title} 4.16.
+link:https://issues.redhat.com/browse/OADP-4075[OADP-4075]
+
+.Data Mover backups work properly in the {product-title} 4.16 cluster
+
+Previously, Velero was using the earlier version of SDK where the `Spec.SourceVolumeMode` field did not exist. As a consequence, Data Mover backups failed in the {product-title} 4.16 cluster on the external snapshotter with version 4.2.
+With this update, external snapshotter is upgraded to version 7.0 and later. As a result, backups do not fail in the {product-title} 4.16 cluster.
+link:https://issues.redhat.com/browse/OADP-3922[OADP-3922]
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12438505[OADP 1.4.0 resolved issues] in Jira.
+
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+
+.Backup fails when checksumAlgorithm is not set for MCG
+
+While performing a backup of any application with Noobaa as the backup location, if the `checksumAlgorithm` configuration parameter is not set, backup fails. To fix this problem, if you do not provide a value for `checksumAlgorithm` in the Backup Storage Location (BSL) configuration, an empty value is added.
+The empty value is only added for BSLs that are created using Data Protection Application (DPA) custom resource (CR), and this value is not added if BSLs are created using any other method.
+link:https://issues.redhat.com/browse/OADP-4274[OADP-4274]
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12438506[OADP 1.4.0 known issues] in Jira.
+
+
+[id="upgrade-notes-1-4-0_{context}"]
+== Upgrade notes
+
+[NOTE]
+====
+Always upgrade to the next minor version. *Do not* skip versions. To update to a later version, upgrade only one channel at a time. For example, to upgrade from {oadp-first} 1.1 to 1.3, upgrade first to 1.2, and then to 1.3.
+====
+
+[id="changes-oadp-1-3-to-1-4_{context}"]
+=== Changes from OADP 1.3 to 1.4
+
+The Velero server has been updated from version 1.12 to 1.14. Note that there are no changes in the Data Protection Application (DPA).
+
+This changes the following:
+
+* The `velero-plugin-for-csi` code is now available in the Velero code, which means an `init` container is no longer required for the plugin.
+
+* Velero changed client Burst and QPS defaults from 30 and 20 to 100 and 100, respectively.
+
+* The `velero-plugin-for-aws` plugin updated default value of the `spec.config.checksumAlgorithm` field in `BackupStorageLocation` objects (BSLs) from `""` (no checksum calculation) to the `CRC32` algorithm. For more information, see link:https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/release-1.10/backupstoragelocation.md[Velero plugins for AWS Backup Storage Location]. The checksum algorithm types are known to work only with AWS.
```

Comment:
We cannot link to GitHub, per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites.